### ISSUE TITLE
Help (translation issue)

### ISSUE THREAD

**Issue by alevtina-karashokova** [crosswire/xiphos#1122]
Hello! 
I'm translating the Xiphos interface and it's not quite clear for me what _Outline_ in the Prayer List module means (_Outlined topic, Book/Chapter outline_). Could someone help me?

**Comment by LAfricain** [comment]
In French we translated with the word "list".

**Comment by alevtina-karashokova** [comment]
I.e. 'Create a new book/chapter' outline means 'create a new book/chapter list'?

**Comment by LAfricain** [comment]
I Translate the French to english with other words: "Create a new frame organized by Book/Chapter of the Bible"
In French for outline we use 3 different words. In which language are you translating? Thank you for your work!

**Comment by alevtina-karashokova** [comment]
I'm translating into Russian. Thank you for your help!

**Comment by alevtina-karashokova** [comment]
Yes, the translated segments can be helpful. Mostly I use the Russian Synodal Bible.

**Comment by karlkleinpaste** [comment]
It appears that this has been answered adequately.
